lib/api/controllers/securityController.js

```diff
@@ -385,3 +388,31 @@ module.exports = function SecurityController (kuzzle) {
   };
 
 };
+
+/*eslint vars-on-top: 0*/
```

Comment:
Question: instead of having an eslint guard, why do you not define `function createOrReplaceRole (...)`?